src/components/InventoryHostStaleness/HostStalenessCard.js

```diff
@@ -116,14 +174,27 @@ const HostStalenessCard = () => {
             </p>
             <Flex className="pf-u-mt-md">
               <Title headingLevel="h6">System configuration </Title>
-              <a
-                role="button"
-                onClick={() => {
-                  setEdit(!edit);
-                }}
-              >
-                Edit
-              </a>
+              {canModifyHostStaleness ? (
+                <Button
+                  variant="link"
+                  role="button"
+                  onClick={() => {
+                    setEdit(!edit);
```

Comment:
nit-pick: maybe some more explicit naming for the state variable would be better like `isEditing`